query-ranges-to-vnodes.hh

```diff
@@ -0,0 +1,47 @@
+/*
+ * Copyright (C) 2021-present ScyllaDB
+ *
+ * Modified by ScyllaDB
+ */
```

Comment:
You can drop the "Modified by ScyllaDB", we thought it was needed for Apache-derived code, but even that is unnecessary.